backend/apps/ai/common/create_chunks_and_embeddings.py

```
@@ -0,0 +1,56 @@
+) -> list[Chunk]:
+    """
+    Creates chunks and embeddings from given texts using OpenAI embeddings.
+
+    Args:
+        all_chunk_texts (list[str]): List of text chunks to embed.
+        content_object: The object to associate the chunks with.
+        openai_client: Initialized OpenAI client instance.
+
+    Returns:
+        list[Chunk]: List of Chunk instances (not saved).
+    """
```

Ruff (0.11.9)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)